packages/web_benchmarks/lib/src/browser.dart

```diff
@@ -23,15 +22,15 @@ class ChromeOptions {
     this.url,
     this.windowWidth = 1024,
     this.windowHeight = 1024,
-    this.headless,
+    this.headless = true,
```

Comment:
Why have you changed the semantics of this parameter from "must not be null" (the documented requirement) to "defaults to true"? The behavior-preserving change is to make it `required`.

Reply:
Sure, I considered having it headless as a default value, since this is generally used for benchmarking, but instead, I now think it should be defaulted to false. changed to required as of now